src/content/en/2020/markup.md

```diff
@@ -352,13 +345,11 @@ Standard elements are those that are or were part of the HTML specification. Whi
 <figcaption>{{ figure_link(caption="Low probabilities of finding a given element in pages of the sample.", sheets_gid="184700688", sql_file="pages_element_count_by_device_and_element_type_present.sql") }}</figcaption>
 </figure>
 
-{# TODO(authors): Interpet results. #}
-
 ### Custom elements
 
 The 2019 edition of the Web Almanac handled [custom elements](../2019/markup#custom-elements) by discussing several non-standard elements. This year, we found it valuable to have a closer look at custom elements. How did we determine these? Roughly by looking at [their definition](https://html.spec.whatwg.org/multipage/custom-elements.html#custom-elements-core-concepts), notably their use of a hyphen. Let's focus on the top elements, in this case elements used on ≥1% of all URLs in the sample:
 
-{# TODO(authors, analysts): Clarify occurrences and percentages _of what_. Pages? Elements? And for desktop or mobile? #}
+{# TODO(authors, analysts): Clarify occurrences and percentages _of what_. Pages? Elements? #}
```

Comment:
Background (for removing part of the TODO): We described in the intro how we usually look at the mobile set. (Open to calling this out again if we feel it needs a reminder in between.)